ui/src/views/Tasks/TaskGroup/index.jsx

```diff
@@ -96,9 +96,10 @@ const updateTaskGroupIdHistory = id => {
         kind: {
           $in: ACTIONS_JSON_KNOWN_KINDS,
         },
-        context: {
-          $or: [{ $size: 0 }, { $size: 1 }],
-        },
+        $or: [
+          { context: { $size: 0 } },
+          { context: { $size: 1 } }
+        ],
```

Comment:
There's another instance of this on line 524 that likely needs fixed too then

Reply:
oh! Totally right, thank you.!